ci_build/azure_pipelines/unit_test.yml

```diff
@@ -56,7 +56,7 @@ stages:
         # tf 1.12
         python_versions: [3.6']
         tf_versions: ['1.12.3']
-        onnx_opsets: ['']
+        onnx_opsets: ['9']
```

Comment:
Looking at the tests, I don't think this flag is working properly.  Might want to fix later.